pkg/cluster/api/pdapi.go

```diff
@@ -661,6 +662,21 @@ func (pc *PDClient) GetReplicateConfig() ([]byte, error) {
 	})
 }
 
+// GetLocationLabels gets the replication.location-labels config from pd server
+func (pc *PDClient) GetLocationLabels() ([]string, error) {
+	config, err := pc.GetReplicateConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	rc := pdconfig.ReplicationConfig{}
+	if err := json.Unmarshal(config, &rc); err != nil {
+		return nil, errors.Annotate(err, "unmarshal replication config")
```

Comment:
Do we need to attach the raw byte string to the errors? (avoid to ask users fetch data via HTTP API)